avidemux_fr.ts

```diff
@@ -1551,22 +1551,22 @@ Ignorer les informations de timing ?</translation>
     <message>
         <location line="+1"/>
         <source>Save as BMP</source>
-        <translation>Enregistrer comme BMP</translation>
+        <translation>Enregistrer en .BMP</translation>
```

Comment:
BMP means image format here, not filename extension. I don't think we should propagate the sin of seeing them as interchangeable and identical.